### PR TITLE
Note another method to block xmlrpc.php attacks.

### PR DESCRIPTION
**Pull request by sarahg**
**Note:** Please fill out the PR Template to ensure proper processing.

## Summary

**[WordPress Best Practices](https://pantheon.io/docs/wordpress-best-practices#avoid-xml-rpc-attacks)** - Note that updating WP core is another way to protect your site from xmlrpc.php shenanigans.

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6036](https://pr-6036--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `66` | `67` | `1` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-6036` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/24097/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/24097/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6036--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/wordpress-best-practices.md [hunk 1]
```diff
@@ -69,6 +69,13 @@ Hits Vis.     %   Bandwidth Avg. T.S. Cum. T.S. Max. T.S. Data
 
 Pantheon recommends disabling XML-RPC, given the WordPress Rest API is a stronger and more secure method for interacting with WordPress via external services.
 
+
+<Alert type="info" title="Note">
```
Comment (alexfornuto) [inline source/content/wordpress-best-practices.md:73]: For copy editor: remove callout.

File: source/content/wordpress-best-practices.md [hunk 2]
```diff
@@ -69,6 +69,13 @@ Hits Vis.     %   Bandwidth Avg. T.S. Cum. T.S. Max. T.S. Data
...
+
+Pantheon blocked requests to `xmlrpc.php` by default in the [WordPress 5.4.2 core release](/changelog/#changelog/2020/07). If your version of WordPress is older than this, you can block `xmlrpc.php` attacks by simply applying your upstream updates.
```
Comment (alexfornuto) [inline source/content/wordpress-best-practices.md:75]: For copy editor: link to core-updates

**Review by alexfornuto — APPROVED** [review]
Thanks @sarahg!

**Pull request merged**